.taskcluster.yml

```diff
@@ -185,7 +185,7 @@ tasks:
                                 else:
                                     $if: 'isPullRequest'
                                     then:
-                                        - 'assume:repo:github.com/${event.pull_request.base.repo.full_name}:pull-request'
+                                        - 'assume:repo:github.com/${event.pull_request.base.repo.full_name}:${tasks_for[7:]}'
```

Comment:
That's what we have on firefox-android: 

https://github.com/mozilla-mobile/firefox-android/blob/6b5dee07befa178e6a8b5c53fcdcc7bfe9c096ea/.taskcluster.yml#L223